packages/composites/savax-price/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@chainlink/savax-price-adapter",
+  "version": "1.0.32",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
we should update the `new` script as well to set the version to `1.0.0`